### PR TITLE
Rearrange website content

### PR DESCRIPTION
**Pull request by FarnazH**
I removed some tabs from the website and provided the same information on the front page (in a more compact manner).
Feel free to let me know what you think.

### CODE REVIEW

**Comment by fwmeng88** [comment]
The new arrangement looks nice! Thanks! @FarnazH 

Only one minor comment: it seems we need to delete `intro_about.rst` file in `docs` folder as it is no longer need.

**Comment by fwmeng88** [comment]
For the failed tests, if we replace all the **docs** in `tox.ini` with **doc** and they should be resolved. Apart from that, I am happy to merge this PR. 
@FarnazH

File: .gitignore [hunk 1]
```diff
@@ -106,3 +106,6 @@ tags
 
 # jupyter notebook checkpoints
 .ipynb_checkpoints
+
+# version file generated by rob
+procrustes/_version.py
```
Comment (fwmeng88) [inline .gitignore:111]: We should also replace all the **docs** with **doc** in .gitignore.

**Review by fwmeng88 — CHANGES_REQUESTED** [review]

**Comment by FarnazH** [comment]
Sorry, @fwmeng88 I was experimenting with `Roberto` in my forked repo and pushed it to the wrong branch... This PR was meant to only contain the first 2 commits. I will force push to fix it!

File: .gitignore (hunk 1 above)
Comment (FarnazH) [reply .gitignore:111]: Thanks. I will fix it in my future PR.

**Review by fwmeng88 — APPROVED** [review]

**Pull request merged**